hack/jenkins/common.sh

```diff
@@ -176,18 +176,22 @@ echo
 echo ">> Downloading test inputs from ${MINIKUBE_LOCATION} ..."
 gsutil -qm cp \
   "gs://minikube-builds/${MINIKUBE_LOCATION}/minikube-${OS_ARCH}" \
-  "gs://minikube-builds/${MINIKUBE_LOCATION}/docker-machine-driver"-* \
-  "gs://minikube-builds/${MINIKUBE_LOCATION}/e2e-${OS_ARCH}" out
+  "gs://minikube-builds/${MINIKUBE_LOCATION}/e2e-${OS_ARCH}" \
+  out
 
 gsutil -qm cp -r "gs://minikube-builds/${MINIKUBE_LOCATION}/testdata"/* testdata/
 
 gsutil -qm cp "gs://minikube-builds/${MINIKUBE_LOCATION}/gvisor-addon" testdata/
 
+if [[ "${DRIVER}" == "hyperkit" ]]; then
+    gsutil -qm cp "gs://minikube-builds/${MINIKUBE_LOCATION}/docker-machine-driver-hyperkit" out
+    chmod +x out/docker-machine-driver-hyperkit
+fi
```

Comment:
this looks like an unrelated change for this pr - if so, shall we move it to a separate pr ?

Reply:
This replaces removed line 179:

```
"gs://minikube-builds/${MINIKUBE_LOCATION}/docker-machine-driver"-*
```

And moified line 190:

```
chmod +x "${MINIKUBE_BIN}" "${E2E_BIN}" out/docker-machine-driver-*
```

Since now the only aux driver is hyperkit, we have a separate block for downloading the driver. Hopefully this will be removed soon in #21601.